quickstart/examples/joins/README.md

```diff
@@ -0,0 +1,53 @@
+## Inner Join
+
+- The INNER JOIN creates a new result table by combining column values of two tables (table1 and table2) based upon the join-predicate. The query compares each row of table1 with each row of table2 to find all pairs of rows which satisfy the join-predicate.
+- In order to keep state size in check, we recommend using time interval joins.
+- Time interval should be in the format table2.rowtime BETWEEN table1.rowtime - INTERVAL 'first duration' unit AND table1.rowtime + INTERVAL 'second duration' unit (both durations are numbers & units can be SECOND/MINUTE/HOUR)
+
```

Comment:
Same as this
https://github.com/odpf/dagger/pull/214/files#r1049652605

Let's not recommend best practices or configs here. We only need to help the user run a demo example. 

We can explain these things in the future story, where will create a dedicated folder called `Concepts`